src/main/java/nl/tudelft/ti2206/group9/gui/ExternalTicker.java

```diff
@@ -48,7 +51,14 @@ private void renderScene() {
 			final Group entities = renderEntities();
 			GameScreen.addWorld(entities);
 		}
+		
+		GameScreen.addOverlay(renderScore());
+	}
 
+	/**
+	 * @return VBox with score labels
+	 */
+	private VBox renderScore() {
```

Comment:
That is true. I did this because I had added the score and ticks to the scorebox, and the method became too long, therefore I extracted this part.

<review